package.json

```diff
@@ -178,29 +172,20 @@
       {
         "command": "PowerShell.ShowHelp",
         "title": "Get Help for Command",
-        "icon": {
-          "light": "media/resources/light/question.svg",
-          "dark": "media/resources/dark/question.svg"
-        },
+        "icon": "$(question)",
         "category": "PowerShell"
       },
       {
         "command": "PowerShell.Debug.Start",
         "title": "Run",
         "category": "PowerShell",
-        "icon": {
-          "light": "media/resources/light/run.svg",
-          "dark": "media/resources/dark/run.svg"
-        }
+        "icon": "$(run-all)"
```

Comment:
Since <1.56. Could also just use `$(run)` (the single arrow, used next) and instead use `$(debug-line-by-line)` for "Run Selection" which would look like this:
<img width="110" alt="Screen Shot 2022-03-04 at 2 42 07 PM" src="https://user-images.githubusercontent.com/2226434/156851810-a30e9695-6c64-40fc-8226-86b2c41538b9.png">

Reply:
This is the current iteration. It looks more like what we had.